datafusion/physical-expr/src/aggregate/median.rs

```diff
@@ -151,7 +151,16 @@ impl Accumulator for MedianAccumulator {
                 // ignore null values
                 .filter(|v| !v.is_null())
                 .cloned(),
-        )?;
+        );
+
+        if array.is_err() {
```

Comment:
`array.map_err(|e| DataFusionError::Execution("".to_owned()))?;` You may find this syntax more concise